tox.ini

```diff
@@ -1,8 +1,11 @@
 [tox]
-envlist = py36,py37,py38,py39
+envlist = py{3.6,3.7,3.8,3.9}-jsonschema{3,4}
 skip_missing_interpreters = True
 
 [testenv]
+deps =
+    jsonschema3: jsonschema>=3.0.0,<4.0.0
+    jsonschema4: jsonschema>=4.0.0,<5.0.0
```

Comment:
This would only be needed if we want to support `jsonschema` versions 3 **and** 4 (and, in that case, I need to check how to run this in the CI yet).